search/search_quality/sample.cpp

```diff
@@ -74,6 +59,17 @@ void FromJSONObject(json_t * root, string const & field, m2::RectD & result)
   result.setMaxY(maxY);
 }
 
+void FromJSONObject(json_t * root, string const & field, m2::PointD & result)
```

Comment:
I tried but I did not managed to do that. The problem is they could be used in 
template <typename T> void FromJSONObject(json_t * root, string const & field, vector<T> & result);

This template is instantiated in these cpp-unit below several times with different params T and calls FromJSONObject from sample.cpp and from myjansson.hpp.